### ISSUE TITLE
When try to change the month into the calendar through keyboard navigation it loses focus

### ISSUE THREAD

**Issue by Aleksandyr** [IgniteUI/igniteui-angular#1015]
### Description  
Open: 
https://www.infragistics.com/products/ignite-ui-angular/angular/components/calendar.html

### Steps to reproduce  

1. Navigate to the new month with keyboard arrow keys 
![capture](https://user-images.githubusercontent.com/7037740/38814535-5d39e2d4-419a-11e8-92cc-748fcc2fe453.PNG)
2. Press enter in way to change the month

### Result  
The focus leaves the calendar and you are unable the go through the dates by keyboard navigation.

### Expected result  
The focus should remain and you have to be able to navigate through the dates.
